internal/services/monitor/monitor_scheduled_query_rules_alert_resource.go

```diff
@@ -385,17 +396,20 @@ func expandMonitorScheduledQueryRulesAlertingAction(d *pluginsdk.ResourceData) *
 	alertAction := expandMonitorScheduledQueryRulesAlertAction(alertActionRaw)
 	severityRaw := d.Get("severity").(int)
 	severity := strconv.Itoa(severityRaw)
-	throttling := d.Get("throttling").(int)
+	throttling, throttlingOk := d.GetOk("throttling")
```

Comment:
This get and the if conditional below could be compressed together - see my comment for line 411